### ISSUE TITLE
`no-invalid-meta` rule crashes on meta tag without content attribute

### ISSUE THREAD

**Issue by jalandis** [ember-template-lint/ember-template-lint#1080]
Getting the following error related to the `no-invalid-meta` rule. 

```
/workspace/ember-app/app/app/templates/head.hbs
  -:-  error  contentAttrValue.match is not a function  undefined
TypeError: contentAttrValue.match is not a function
    at NoInvalidMeta.ElementNode (/workspace/ember-app/app/node_modules/ember-template-lint/lib/rules/no-invalid-meta.js:53:30)
    at /workspace/ember-app/app/node_modules/ember-template-lint/lib/rules/base.js:292:21
    at visitNode (/workspace/ember-app/app/node_modules/@glimmer/syntax/dist/commonjs/es5/lib/traversal/traverse.js:74:18)
    at visitArray (/workspace/ember-app/app/node_modules/@glimmer/syntax/dist/commonjs/es5/lib/traversal/traverse.js:148:22)
    at visitKey (/workspace/ember-app/app/node_modules/@glimmer/syntax/dist/commonjs/es5/lib/traversal/traverse.js:128:9)
    at visitNode (/workspace/ember-app/app/node_modules/@glimmer/syntax/dist/commonjs/es5/lib/traversal/traverse.js:92:13)
    at traverse (/workspace/ember-app/app/node_modules/@glimmer/syntax/dist/commonjs/es5/lib/traversal/traverse.js:185:5)
    at transform (/workspace/ember-app/app/node_modules/ember-template-recast/src/index.js:36:3)
    at Linter.verify (/workspace/ember-app/app/node_modules/ember-template-lint/lib/linter.js:173:9)
    at lintFile (/workspace/ember-app/app/node_modules/ember-template-lint/bin/ember-template-lint.js:18:17)

✖ 1 problems (1 errors, 0 warnings)
```

**Comment by bmish** [comment]
Hey @MelSumner, looks like a `<meta>` tag without a `content` attribute crashes the `no-invalid-meta` rule. Should the rule ignore that situation or report a violation?

**Comment by paulcwatts** [comment]
I also noticed this error happens if the `content` parameter is specified using curlies, like:

```
<meta property="og:title" content={{this.title}} />
```

**Comment by bmish** [comment]
I opened a PR to fix these issues in #1087.
